andela/ah-the-jedi-frontend
# #165305216 Configure test environment

### What does this PR do?
Setup environment for running tests

### Description of Task(s) to be completed?
- install and configure jest
- install and configure enzyme

### How should this be manually tested?

Clone this repository on local machine and CD into project folder
`$ git clone git@github.com:andela/ah-the-jedi-frontend.git`
`$ cd ah-the-jedi-frontend`

Check out to the PR's branch
`$ git checkout ch-setup-test-env-165305216`

Run tests
`npm test`

### What are the relevant PT stories?
[#165305216](https://www.pivotaltracker.com/story/show/165305216)

### Screenshot
![image](https://user-images.githubusercontent.com/11965186/58480359-e3321880-8162-11e9-9348-a58f8882eb02.png)
